packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -357,6 +357,12 @@ export class StripeWebhookHandler extends StripeHandler {
           invoice,
           error.longMessage
         );
+
+        this.log.error('handleCreditNoteEvent', {
+          invoiceId: invoice.id,
+          message: 'Paypal refund refused.',
```

Comment:
Q: Is this user facing, and if so, does it need to be localized? 
Also, should we change it to PayPal to appease the copyright gods?

Reply:
This is not user facing, its going into our logging in big query.